opennms-doc/guide-admin/src/asciidoc/text/telemetryd/troubleshooting.adoc

```diff
@@ -0,0 +1,50 @@
+
+[[telemetryd-troubleshooting]]
+=== Troubleshooting
+
+Configuring _telemetryd_ involves a few components.
+This section gives a few troubleshooting hints to verify if the configuration and the network setup is correct.
+
+.Ensure the Java process is listening on UDP port 50000
+[source]
+----
+ss -lnp sport = :50000
+
+Netid State  Recv-Q Send-Q Local Address:Port Peer Address:Port
+udp   UNCONN 0      0      *:50000                           *:* users:(("java",pid=585,fd=1226))
+----
+
+.List all firewall rules to ensure it is not blocking incoming traffic
+[source]
+----
+sudo iptables -L
+----
+
+.Example to set a firewall rule with firewalld to allow traffic on UDP port 50000
+[source]
+----
+firewall-cmd --add-port=50000/udp --zone=public
+----
+or
+
+.Example to set a firewall rule with iptables to allow traffic on UDP port 50000
+[source]
+----
+iptables -I INPUT -p udp -m udp --dport 50000 -j ACCEPT
+----
+
+.Check if packets can be written to the internal queue for the JTI protocol
+[source]
+----
+tail -f ${OPENNMS_HOME}/logs/telemetryd.log | grep "JmsConsumer[OpenNMS.Sink.Telemetry.JTI]"
+----
+
+.Check the telemetryd.log if you get authentication errors writing to the JMS queue
```

Comment:
This error will not show up in `telemetryd.log`. It will show up on the Minion.

I think this case should be covered in a general "Verify your Minion" page instead of here, since this will also affect heartbeat, syslog, traps, etc...